sdk/storage/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure Storage libraries for .NET

```

Comment:
client libraries?

Reply:
It includes management libraries as well so I left it just "libraries"

Reply:
:( makes sense